data/json/scenarios.json

```diff
@@ -378,7 +378,7 @@
     "name": "Challenge - Island Prison",
     "points": -6,
     "description": "You were at a high-security prison right before the Cataclysm.  You managed to make it outside the inner walls…  Too bad it's located on a remote island, and now you need to find out how to escape it too.",
```

Comment:
```suggestion
    "description": "You were in a high-security prison right before the Cataclysm.  You managed to make it outside the inner walls…  Too bad it's located on a remote island, and now you need to find out how to escape it too.",
```

Reply:
Honestly this should be completely rewritten, as in this particular variant player starts inside the prison itself.

Reply:
```suggestion
    "description": "You were in a high-security prison right before the Cataclysm.  Unfortunately, you're still in it, so you need to figure out how to escape.  (It's located on a remote island to boot; you'll eventually need to escape that also.)"
```